tests/integration/infra.py

```diff
@@ -24,116 +24,19 @@
-    is no crash dump to attach to this is going to
-    be an empty drgn.Program.
-    """
-    prog = drgn.Program()
-    if not dump_exists():
-        return prog
-    prog.set_core_dump(DUMP_PATH)
-    load_debug_info(prog, [VMLX_PATH, MODS_PATH])
-    return prog
-
-
-TEST_PROGRAM = setup_target()
-TEST_REPL = REPL(TEST_PROGRAM, list(sdb.get_registered_commands().keys()))
-
-
-def repl_invoke(cmd: str) -> int:
-    """
-    Accepts a command/pipeline in string form and evaluates
-    it returning the exit code of the evaluation emulating
-    the SDB repl.
-    """
-    assert TEST_PROGRAM
-    return TEST_REPL.eval_cmd(cmd)
-
-
-def sdb_invoke(objs: Iterable[drgn.Object], line: str) -> Iterable[drgn.Object]:
-    """
-    Dispatch to sdb.invoke, but also drain the generator it returns, so
-    the tests can more easily access the returned objects.
-
-    This method is preferred over repl_invoke() when the test wants to
-    do fancier checks by mocking a few objects that are later passed
-    down to the pipeline. Other scenarios include but are not limited
-    to testing that specific exceptions are thrown or analyzing internal
-    state of objects that is not part of the output in stdout.
-    """
-    assert TEST_PROGRAM
-    return list(sdb.invoke(TEST_PROGRAM, objs, line))
-
-
-def slurp_output_file(modname: str, cmd: str) -> str:
-    """
-    Given a module name and a command, find the output file
-    and return all of its contents as a string.
-    """
-    return Path(f"{TEST_OUTPUT_DIR}/{modname}/{cmd}").read_text()
-
-
-def generate_output_for_commands(cmds: List[str], dirpath: str) -> None:
-    """
-    Takes a list of SDB commands in string form, invokes them in the
-    context of the current crash dump/sdb.REPL, and stores their output
-    in the directory specified, each under a different file.
-
-    Note: Keep in mind that if the directory specified exists then
-    it will be removed together with all of its contents.
-    """
-    assert TEST_PROGRAM
-    if os.path.exists(dirpath):
-        shutil.rmtree(dirpath)
-    os.makedirs(dirpath)
-    for cmd in cmds:
-        with open(f"{dirpath}/{cmd}", 'w') as f:
-            with redirect_stdout(f):
-                repl_invoke(cmd)
-
-
-def generate_output_for_test_module(modname: str) -> None:
-    """
-    Generates the regression output for all the commands of
-    a test module given  module name. The assumption for this
-    to work automatically is that for the given modname "mod"
-    there exist a test module under test.integration named
-    test_mod_generic which has a list of commands in string
-    form called CMD_TABLE.
-    """
-    test_mod = import_module(f"tests.integration.test_{modname}_generic")
-    generate_output_for_commands(
-        test_mod.CMD_TABLE,  # type: ignore[attr-defined]
-        f"{TEST_OUTPUT_DIR}/{modname}")
-    print(f"Generated regression test output for {modname}...")
+PRIMARY = "primary"
+ALTERNATE = "alternate"
```

Comment:
Instead of `PRIMARY` and `ALTERNATE` let's go ahead and have a list of crash dumps automatically detected by the target `DATA_DIR`. For example this would currently look like this with the two crash dumps that we have:
```
DATA_DIR/dump-<date 1>/{dump, mods, vmlinux, regression_output}
DATA_DIR/dump-<date 2>/{dump, mods, vmlinux, regression_output}
```

Keeping this structure as is makes it easy to add new crash dumps in the future.